src/ui/components/Settings.tsx

```diff
@@ -15,7 +15,10 @@ tabs[SettingsTabs.OPTIONS] = (props: SettingsProps) => (
             item: SettingsTabs.TRACKS,
             label: props.data.getTranslation('Quality'),
             info: `${
-              props.data.activeTrack ? props.data.activeTrack.width : ''
+              props.data.activeTrack ?
```

Comment:
Nested condition shorthands are a bit confusing, could this be better structured?

Reply:
as i already mentioned in a earlier reply, i'm not very familiar with TS or JS. if it would be python or rust code, i would definitely choose the most pleasant and idiomatic expression. but on the java  script side i'm unfortunately hardly able to keep my head above water. ;)

if you don't mind, i would prefer, that you simply modify the suggested changes to the better.

i know, it's a little bit of annoying work, but at the end  it's definitely better to keep the code base clean and in exemplary style, which i'm simply not capable to serve in this particular programming language without spending huge amounts of time to study and learn the most basic expressive possibilities.